spec/controllers/external_authors_controller_spec.rb

```diff
@@ -114,9 +114,9 @@
         end
 
         context "when updating preferences" do
-          xit "renders edit template with a success message for orphaning and an error for preferences" do
+          it "renders edit template with a success message for orphaning and an error for preferences" do
             allow_any_instance_of(ExternalAuthor).to receive(:update_attributes).and_return(false)
-            put :update, invitation_token: invitation.token, id: external_author.id, imported_stories: "orphan", do_not_email: true
+            put :update, invitation_token: invitation.token, id: external_author.id, imported_stories: "orphan"
```

Comment:
I think this should be fixed the _other_ way -- that is, by making sure we can pass do_not_email parameters when they're correctly-formatted. I think that would look like `external_author: { do_not_email: true }`